com.maplerain.telsis_translator_flutter.yml

```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: telsis_translator_flutter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  # telsis_translator_flutter
+  - name: telsis_translator_flutter
+    buildsystem: simple
+    build-commands:
+      - mv com.maplerain.telsis_translator_flutter.png /app/share/icons/hicolor/256x256/apps/
+      - mv com.maplerain.telsis_translator_flutter.desktop /app/share/applications/
+      - mv com.maplerain.telsis_translator_flutter.metainfo.xml /app/share/metainfo/
```

Comment:
```suggestion
      - install -Dm644 com.maplerain.telsis_translator_flutter.png -t /app/share/icons/hicolor/256x256/apps/
      - install -Dm644 com.maplerain.telsis_translator_flutter.desktop -t /app/share/applications/
      - install -Dm644 com.maplerain.telsis_translator_flutter.metainfo.xml -t /app/share/metainfo/
```

Reply:
I am stupid for not understanding the `install` command.